x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_significant_events_view/edit_significant_event_flyout.tsx

```diff
@@ -86,12 +78,13 @@ export const EditSignificantEventFlyout = ({
 
                 telemetryClient.trackSignificantEventsCreated({
                   count: 1,
-                  count_by_feature_type: {
-                    system: 0,
-                    ...(data.query.feature && {
-                      [(data.query.feature as GeneratedSignificantEventQuery['feature'])!.type]: 1,
-                    }),
-                  },
+                  count_by_feature_type: !data.query.feature
```

Comment:
While we're in this area, can we fix the types of the events that report feature by types to use `FeatureType` instead of `string` ? ([example](https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/streams_app/public/telemetry/types.ts#L104))